client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py

```diff
@@ -2,64 +2,47 @@
 dolbyio_rest_apis.streaming.models.subscribe_token
 ~~~~~~~~~~~~~~~
 
-This module contains the models used by the Subscribe Token models.
+This module contains the models used by the Subscribe Token module.
 """
 
-from typing import List
-from dolbyio_rest_apis.core.helpers import get_value_or_default
+from dataclasses import dataclass, field
+from dataclasses_json import LetterCase, dataclass_json
+from dolbyio_rest_apis.streaming.models.publish_token import TokenEffectiveSettings, TokenStreamName
 
-class SubscribeTokenStream(dict):
-    """The :class:`SubscribeTokenStream` object, which represents a Subscribe Token Stream."""
-
-    def __init__(self, dictionary: dict):
-        dict.__init__(self, dictionary)
-
-        self.stream_name = get_value_or_default(self, 'streamName', None)
-        self.is_regex = get_value_or_default(self, 'isRegex', False)
-
-class SubscribeToken(dict):
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class SubscribeToken:
```

Comment:
missing tracking property returned in read token api
https://docs.dolby.io/streaming-apis/reference/subscribetokenv1_readtoken

Reply:
✅